bot2.js

```diff
@@ -11,158 +11,186 @@ const config = require("./config.json");
 // config.token contains the bot's token
 // config.prefix contains the message prefix.
 
+
 client.on("ready", () => {
-  // This event will run if the bot starts, and logs in, successfully.
-  console.log(`Bot has started, with ${client.users.size} users, in ${client.channels.size} channels of ${client.guilds.size} guilds.`); 
-  // Example of changing the bot's playing game to something useful. `client.user` is what the
-  // docs refer to as the "ClientUser".
-  client.user.setActivity(`!help for help. Bot serving ${client.users.size} users, in ${client.channels.size} channels`);
 
+    // This event will run if the bot starts, and logs in, successfully.
+    console.log(`Bot has started, with ${client.users.size} users, in ${client.channels.size} channels of ${client.guilds.size} guilds.`);
+
+    // Example of changing the bot's playing game to something useful. `client.user` is what the
+    // docs refer to as the "ClientUser".
+    client.user.setActivity(`!help for help. Bot serving ${client.users.size} users, in ${client.channels.size} channels`);
 });
 
+
+// Guild and Server are synonymous
 client.on("guildCreate", guild => {
-  // This event triggers when the bot joins a guild.
-  console.log(`New guild joined: ${guild.name} (id: ${guild.id}). This guild has ${guild.memberCount} members!`);
-  // client.user.setActivity(`Serving ${client.guilds.size} servers`);
-  // message.channel.send(' \n \
-  //     Welcome, this bot provides content from the HSTR Project. Below are commands to retrieve information. To read the guide and get advice from the community, join here: https://discord.gg/yPchf8r \n \
-  //     Bot is maintained by TheNo0b#9329, directly contact or tag in the Discord server for bugs/suggestions \
-  //     **!teamslist**: Display the full reccomended HSTR teams list \n \
-  //     **!jtr**: display the P1 JTR battlecard.\n \
-  //     **!p1jedi**: display the P1 Jedi battlecard \n \
-  //     **!p2jedi**: display the P2 Jedi battlecard \n \
-  //     **!phoenix**: display the P2 Phoenix battlecard \n \
-  //     **!leia**: display the P2 Machine Gun Leia battlecard \n \
-  //     **!churchofnute**: display the P2 Church of Nute battlecard \n \
-  //     **!chexmix**: display the P3 Chex Mix battlecard \n \
-  //     **!yolorolo**: display the P3 YOLO ROLO battlecard \n \
-  //     **!nihilusmatrix**: display matrix to understand how Nihilus attacks \n \
-  //     **!nightsisters**: display the P4 Nightsisters battlecard (strategy applicable to P2 and P3 too) \n \
-  //     **!p1 [number]**: Convert damage score to percent of boss health in P1 \n \
-  //     **!p2 [number]**: Convert damage score to percent of boss health in P2 \n \
-  //     **!p3 [number]**: Convert damage score to percent of boss health in P3 \n \
-  //     **!p4 [number]**: Convert damage score to percent of boss health in P4 \n \
-  //     ');
```

Comment:
Also code bloat, decided to just get rid of it. If we want to show the help message when the bot is added to a server in the future, we can do that. Would probably need to set this information on the side.